pkgs/by-name/nu/numworks-epsilon/package.nix

```diff
@@ -47,9 +53,28 @@ stdenv.mkDerivation rec {
     mkdir -p $out/bin
     cp -r ./output/release/simulator/linux/* $out/bin/
 
+    # To build this svg:
+    # copy white path from ion/src/simulator/assets/icon_mask.svg
+    # paste into ion/src/simulator/assets/logo.svg and change fill to #edb14b
```

Comment:
We can easily do this in shell code.

Reply:
I'm not sure that we should in case these logos change.  The layers aren't labeled so it would be very easy for this logic to break.

Reply:
Then we would need to update the hash, too. Can't we just replace the old color code? is that unique?

Reply:
Which hash would need to be updated?

Reply:
I've added a commit that builds the logo from source.